mypy/util.py

```diff
@@ -746,15 +746,15 @@ def format_error(
 
 def is_typeshed_file(file: str) -> bool:
     # gross, but no other clear way to tell
-    return 'typeshed' in os.path.abspath(file).split(os.sep)
+    return 'typeshed' in pathlib.Path(file).absolute().parts
```

Comment:
I expect this function to be used rather frequently, so I don't think making a new pathlib.Path just to throw it away makes sense here, perhaps you can use `os.path.commonpath` instead?

Reply:
Thanks